app/main.py

```diff
@@ -1,8 +1,23 @@
 class Person:
-    # write your code here
-    pass
+    people: dict = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    new_people = []
+    for person in people:
+        new_people.append(Person(person["name"], person["age"]))
+
+    for person_new in people:
```

Comment:
The loop should iterate over `new_people` instead of `people`. The current implementation uses the original `people` list, which might not have the `Person` instances needed for setting relationships.